lib/auth/index.jsx

```diff
@@ -146,7 +147,16 @@ export class Auth extends Component {
           {isCreatingAccount && (
             <div className="terms">
               By creating an account you agree to our
-              <a href="http://simplenote.com/terms/">Terms of Service</a>.
+              <a
+                href="http://simplenote.com/terms/"
```

Comment:
Might as well make it `https`:

```suggestion
                href="https://simplenote.com/terms/"
```